tests/test_file_download.py

```diff
@@ -191,7 +191,7 @@ def test_file_not_found_locally_and_network_disabled(self):
                 )
 
     def test_private_repo_and_file_cached_locally(self):
-        api = HfApi(endpoint=ENDPOINT_STAGING)
+        api = HfApi(endpoint=ENDPOINT_STAGING, token=TOKEN)
```

Comment:
Why was this change needed here? 
In both calls below, `token=TOKEN` is passed to `create_repo` and `upload_file`. If test has failed for some auth issue, maybe it's really a bug no? (and if no error was raised, we can remove this change)